### PR TITLE
Rejig in Figma, and add px 

### PR DESCRIPTION
**Pull request by CCAyl**
Rejigged some things so that the tokens look nice within figma, but still have the same object structure as before. now our values have px in them by default.

i ran the dictionary build manually/locally, since we still need to fix up that action

### CODE REVIEW

File: data/tokens.json [hunk 1]
```diff
@@ -389,7 +325,7 @@
       "caption": {
         "type": "typography",
         "value": {
-          "fontFamily": "{typography.family.base}",
+          "fontFamily": "{typography.fontFamily.mulish}",
```
Comment (iryanclarke) [inline data/tokens.json:328]: Do we want to explicitly call this Mulish? I'm thinking maybe we abstract it out to `fontFamily.sansSerif` or a more generic name, so we could theoretically change things in the repo without having to re-do instances within the codebase if we delete this variable

File: data/tokens.json [hunk 2]
```diff
@@ -2,281 +2,217 @@
   "global": {
     "colors": {
       "green": {
-        "800": {
-          "value": "#244C38",
-          "type": "color"
-        },
-        "700": {
-          "value": "#275E43",
+        "100": {
```
Comment (iryanclarke) [inline data/tokens.json:5]: This re-jigging just to visually change the order within Figma Tokens?

File: data/tokens.json (hunk 1 above)
Comment (CCAyl) [reply data/tokens.json]: works for me, updated

File: data/tokens.json (hunk 2 above)
Comment (iryanclarke) [reply data/tokens.json:5]: Interesting! Mystery magic I guess

**Review by iryanclarke — APPROVED** [review]
Woohoo! Looks great

**Review by alexandra-lim — APPROVED** [review]
Looks great in the plugin too!

**Pull request merged**